### DISCUSSION TITLE
CPF mask

### DISCUSSION THREAD

**Discussion by joaopaulozz** [RensTillmann/super-forms#53] [Q&A] [answered]
Hi guys,
first I would like to congratulate you on a great plugin.
But, my question is, I would like to know if I can validate a CPF when the user types, validate if the CPF is existing, not only if it has 11 numbers

Att, Joao Paulo

**Comment by RensTillmann** [comment, marked as answer]
Hi, this isn't possible, you would have to write some custom code, and validate it on some sort of API that can validate it.

**Reply by joaopaulozz** [reply]
ok
